h2o-genmodel/src/test/java/hex/genmodel/algos/ensemble/StackedEnsembleMultinomialMojoTest.java

```diff
@@ -0,0 +1,72 @@
+package hex.genmodel.algos.ensemble;
+
+import com.google.common.io.ByteStreams;
+import hex.genmodel.ModelMojoReader;
+import hex.genmodel.MojoReaderBackend;
+import hex.genmodel.easy.EasyPredictModelWrapper;
+import hex.genmodel.easy.RowData;
+import hex.genmodel.easy.prediction.MultinomialModelPrediction;
+import org.junit.Test;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+import static org.junit.Assert.*;
+
+public class StackedEnsembleMultinomialMojoTest {
+
+    @Test
+    public void testPredictMultinomialProstate() throws Exception {
+
+        StackedEnsembleMojoModel mojo = (StackedEnsembleMojoModel) ModelMojoReader.readFrom(new ClasspathReaderBackend());
+        assertNotNull(mojo);
+
+        EasyPredictModelWrapper wrapper = new EasyPredictModelWrapper(mojo);
+
+       MultinomialModelPrediction pred = (MultinomialModelPrediction) wrapper.predict(new RowData() {{
+            put("CAPSULE", "0");
+            put("AGE", "65");
+            put("DPROS", "2");
+            put("DCAPS", "1");
+            put("PSA", "1.4");
+            put("VOL", "0");
+            put("GLEASON", "6");
+        }});
+
+        assertEquals(1, pred.labelIndex);
+        assertEquals("1", pred.label);
+        assertArrayEquals(new double[]{0.006592327, 0.901237, 0.09217069}, pred.classProbabilities, 1e-5);
+
+        System.out.println("Has RACE: " + pred.label);
```

Comment:
What is the purpose of the print statements?

Reply:
No purpose. I can remove.